src/cassim.erl

```diff
@@ -35,11 +37,24 @@
     set_security/3
 ]).
 
+-export([
+    migrate_database/1,
+    migrate_databases/0
+]).
+
 
 -include_lib("couch/include/couch_db.hrl").
 
 
+is_active() ->
+    is_enabled() andalso metadata_db_exists().
+
+
 is_enabled() ->
+    config:get("cassim", "enable", "false") == "true".
```

Comment:
How about `config:get_boolean/3` here? It will return atom either `true` or `false` so there will be no additional comparison with "true" string.